scripts/deblur

```diff
@@ -12,37 +12,42 @@ import click
 from os import listdir, makedirs, mkdir
 from os.path import join, isfile, exists
 from shutil import rmtree
+import logging
 
 from skbio.parse.sequences import parse_fasta
 from skbio.util import remove_files
-from biom.util import biom_open, HAVE_H5PY
 
 from deblur.deblurring import deblur
 from deblur.workflow import (launch_workflow, trim_seqs, dereplicate_seqs,
                              remove_artifacts_seqs,
                              multiple_sequence_alignment,
                              remove_chimeras_denovo_from_seqs,
-                             generate_biom_table,
                              split_sequence_file_on_sample_ids_to_files,
-                             merge_otu_tables,
-                             build_index_sortmerna)
+                             build_index_sortmerna,
+                             get_files_for_table,
+                             create_otu_table,
+                             start_log)
+
+
+__version__ = "0.9.2.1"
```

Comment:
we enable the users to run only these parts of the pipeline as well (i.e. just to join fasta files to a single biom table)